src/Client/TrapHandle.php

```diff
@@ -220,4 +215,9 @@ private function haveToSend(): bool
 
         return true;
     }
+
+    public function __destruct()
```

Comment:
Can we sort methods like:

```
public/private constructor, destructor
public static
public magic methods
public from interfaces
public without interfaces
protected, private
```

The main idea: constructor is always first; desrtructor is always after the constructor

Reply:
It's OK to move destructor to other magic methods, but constructor must be first